### PR TITLE
Feat add omnitracking interact content event

### PR DESCRIPTION
**Pull request by danielbento92**
## Description

- Added interact content event mapping to omnitracking;

### Dependencies

#501.

## Checklist

- [x] The commit message follows our guidelines
- [x] Tests for the respective changes have been added
- [x] The code is commented, particularly in hard-to-understand areas
- [x] The labels and/or milestones were added

## Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

File: packages/core/src/analytics/integrations/Omnitracking/definitions.js
Comment (ruifcnunes) [inline packages/core/src/analytics/integrations/Omnitracking/definitions.js:558]: `[Omnitracking] - Event ${data.event} requires at **least** the properties "contentType" and "interactionType"`

Also just a nitpick: add a blank line between the const and the if (line 555)

Comment (danielbento92) [reply packages/core/src/analytics/integrations/Omnitracking/definitions.js:558]: I spotted this typo and this break line on next task, then I resolved in next PR.

But this branch will need rebase, so I will fix this, before the merge.

File: packages/core/src/analytics/integrations/Omnitracking/definitions.js
Comment (boliveira) [inline packages/core/src/analytics/integrations/Omnitracking/definitions.js:566]: As you have the `properties` variable available, you could use that instead of using `data.properties`.

Comment (danielbento92) [reply packages/core/src/analytics/integrations/Omnitracking/definitions.js:566]: Same as previous item. O Spotted this on scroll task, then I resolve there.

I'm waiting for rebase to fix this and get branch ready to merge.

Comment (boliveira) [reply packages/core/src/analytics/integrations/Omnitracking/definitions.js:566]: Alright! Closing it...

**Review by ruifcnunes — APPROVED** [review]

**Pull request merged**
